sdk/ml/azure-ai-ml/azure/ai/ml/operations/_compute_operations.py

```diff
@@ -416,6 +421,26 @@ def list_sizes(self, *, location: Optional[str] = None, compute_type: Optional[s
             ]
         return [VmSize._from_rest_object(item) for item in size_list.value]
 
+    @distributed_trace
+    @monitor_with_activity(ops_logger, "Compute.enablesso", ActivityType.PUBLICAPI)
+    @experimental
+    def enable_sso(self, *, name: str, enable_sso: Optional[bool] = True):
```

Comment:
"Optional" in Python means that the value can be None, but I think in this case you mean that a value doesn't necessarily have to be provided by the user. Instead, you can just do `enable_sso: bool = True`.